### ISSUE TITLE
Add fake sentence generator with lorem text.

### ISSUE THREAD

**Issue by thedevsbuddy** [PinkFrojd/mock_data#16]
I like the concept and I am using this package but it will be awesome to be able to generate sentences by word count.

**Comment by dinkopehar** [comment]
Hello @thedevsbuddy 

Thank you for taking an interest in this project.
~Similar feature to your request is currently for generating names, which contains predefined list of names.
However, in the feature you are requesting, there would have to be predefined sentences with N number of words in it, but list can't be indefinite (_sentences with 30 words, 40 words, 100 words etc._ ).~

EDIT:
I didn't read correctly title of issue at first, I thought you wanted grammatically valid sentences. I understand now you want Lorem Ipsum generator.
Can you write some ideas or technicalities you want in this features ? I'll look more into this feature in near future and implement it in some way. I'll write about final implementation details here before implementing it.

Thank you.

**Comment by thedevsbuddy** [comment]
Hello @dinko-pehar 

I have an idea for generating Lorem text sentences with word and sentence count I will try to implement it and make a PR so you can check it.

But the idea in my head is a bit hacky for your knowledge what I am thinking is to have a file containing Lorem texts in it and then when calling method like `mockSentance(words: 50, sentence: 2)` we can take words and sentences from the text file and return it from this function also we can add this feature in the `mockString()` method as well because I think currently it returns some randomly generated strings. 

Please let me know what you think about this.

Thank you.

**Comment by dinkopehar** [comment]
@thedevsbuddy 

I agree withyou on creating a lorem text. You could create this in `constants.dart` similar how male and female names are created. This could be the list of "lorem" words _(["ad", "adipisicing","aliqua","aliquip"...])_ with as many different words.

From there, maybe you could expand lorem generator to `text` instead of just `sentence` (_play with the following for an idea https://loremipsum.io)_. 
You could call method `mockIpsum`, `mockText` or something like that with following parameters:
1. `mockText(words: 3)` would result in text with 3 random words , for example: `Lorem ipsum dolor`.
2.  `mockText(sentences: 3)` would result in 3 sentences containing from 5 up to 15 words for each sentence. Something like `Lorem ipsum dolor aliqua sit amet. Aliqua dolor sit amet ipsum dolor. Dolor amet ipsum dolor sit amet ad.`. 
3. `mockText(paragraphs: 3)` would create 3 paragraphs from 3 up to 7 sentences. The output is something like:

> Adipisicing enim aute sit laborum sit. Et excepteur ut minim et laboris sint. Amet nostrud et aliquip occaecat laboris est amet aute esse irure ullamco mollit Lorem. Id anim tempor occaecat sunt pariatur. Incididunt dolor aute ipsum nulla magna consequat eu nostrud non consequat eiusmod anim.
Ipsum et adipisicing consequat est occaecat enim amet elit nostrud magna. Nulla id dolore ipsum culpa magna velit sit officia veniam pariatur fugiat enim laboris tempor. Duis incididunt et fugiat proident esse cillum qui do reprehenderit sint mollit nostrud. Exercitation nulla in Lorem proident veniam enim exercitation sint. Ut aute consectetur pariatur id voluptate id ut deserunt quis excepteur tempor.
Ullamco tempor mollit magna anim veniam amet elit ea cupidatat ex reprehenderit occaecat. Elit eu exercitation eu minim dolor dolor id quis. Ullamco minim consectetur est do minim ea deserunt.

---

I'm not sure about modifying `mockString` though. That method generates random string based on number of wanted characters. Not sure how would we cover cases for `mockString(36)` or some long words from Lorem text. If you think you can do that without breaking the current tests or implementation (_because I would like to keep same calling convention for all versions of this library_), go ahead. But mocking ipsum text would be enough.

Feel free to fork and make PR on this project 🚀 .

**Comment by thedevsbuddy** [comment]
@dinkopehar 

I will try this when I get time and for the concern about the `mockString(36)` we can combine multiple words from Lorem text and trim it to limit for the N character which user passed as parameter. This will be without white space as it should be a single word string as it currently generates.

**Comment by dinkopehar** [comment]
No problem @thedevsbuddy 

This is small library so any contribution from community is more than welcome. If you find time, try to implement what you can. If you find a right way, maybe you can create 2 PRs so we can merge changes faster. Feel free to implement in how it best suits you.
